### DISCUSSION TITLE
Add a 'life-end' event to the Message Component

### DISCUSSION THREAD

**Discussion by vincent-clipet** [primefaces/community#853] [PrimeVue] [answered]
The [Toast ](https://primevue.org/toast/)component currently emits a '**life-end**' event that is fired when its timeout ('life') is reached.
*(this was implemented in commit [57a490ff](https://github.com/primefaces/primevue/commit/57a490ffbd7b9488449ddaf19d3ab86fbc94f024) in response to issue [2989](https://github.com/primefaces/primevue/issues/2989))*

The [Message ](https://primevue.org/message/)component does not fire this '**life-end**' event, despite sharing the same timeout functionality than Toasts, both through their 'life' prop.

**Comment by tugcekucukoglu** [comment, marked as answer]
Why not? We could port it for Message. Could you create an issue for this request?

**Reply by vincent-clipet** [reply]
Sure, done !
https://github.com/primefaces/primevue/issues/5070
